.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -22,8 +22,24 @@
 //  http://groovy-lang.org/style-guide.html
 class common_job_properties {
 
+  static void setBeamSCM(def context, String repositoryName) {
+    context.scm {
+      git {
+        remote {
+          github("apache/${repositoryName}")
```

Comment:
Not sure the best way to go about this, but a bit confusing that `${repositoryName}` is interpolated while `${ghprbPullId}` I think is a Jenkins config level variable. Do I have this right?